elements/bulbs-video/components/revealed.js

```diff
@@ -16,7 +17,6 @@ global.BULBS_ELEMENTS_COMSCORE_ID = '6036328';
 
 let prefixCount = 0;
 function makeGaPrefix () {
-  // ga demands tracker names be alphanumeric
   return `videoplayer${prefixCount++}`;
```

Comment:
Can the prefix counter stuff go into `prepGaEvent`?

So you pass a string into `prepGaEvent` and it makes it unique inside that function?